core/domain/feedback_services.py

```diff
@@ -804,17 +805,18 @@ def _send_feedback_thread_status_change_emails(
                 old_status, new_status)
 
 
-def _check_unique_ids_for_recipients(user_ids, exploration_id, thread_id):
-    user_models = (
-        email_models.FeedbackEmailReplyToIDModel.get_for_multi_user_ids(
+def _check_reply_to_ids_for_recipients(user_ids, exploration_id, thread_id):
+    feedback_email_id_models = (
+        email_models.FeedbackEmailReplyToIdModel.get_for_multi_user_ids(
```

Comment:
I thought I suggested changing this to get_multi_by_user_ids (I think we have a similar pattern elsewhere, with regards to "get ... by ...").